selfdrive/car/hyundai/values.py

```diff
@@ -576,7 +576,8 @@ class CAR(Platforms):
     "GENESIS GV70 1ST GEN",
     [
       HyundaiCarInfo("Genesis GV70 (2.5T Trim) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_l])),
-      HyundaiCarInfo("Genesis GV70 (3.5T Trim) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_m])),
+      HyundaiCarInfo("Genesis GV70 (3.5T Trim) 2022-23", "Highway Driving Assist", car_parts=CarParts.common([CarHarness.hyundai_m])),
+      HyundaiCarInfo("Genesis GV70 (3.5T Trim) 2022-23", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_l])),
```

Comment:
This could make sense, but you're claiming this car is HDA 2 without the ADAS driving ECU?

Reply:
Doesn't look like there is HDA2 for this car. https://www.genesis.com/content/dam/genesis/us/pdf/2022/2022-Genesis%20GV70%20Brochure_vfin.pdf

Reply:
@sshane According to the spec sheet from Genesis, this car (2022 GV70 3.5T AWD) comes with Highway Driving Assist II: https://www.genesis.com/content/dam/genesis/us/com/pdf/2024/2024%20GV70%20Specs%20Sheet_vfin.pdf

![image](https://github.com/commaai/openpilot/assets/47793918/804df75c-01a5-4cc8-a200-53d0857a40a2)

Reply:
That's the 2024 model though, check 2022 and 2023, they both say both trims just have HDA (no 2)

Reply:
If it helps, here is the window sticker for my 2022 Genesis GV70 2.5t Sport prestige that has HDA2. 
![IMG_1104](https://github.com/commaai/openpilot/assets/114946832/65d6a658-c3b0-421d-895f-d0c15b230aca)